src/app/app.component.html

```diff
@@ -1,5 +1,4 @@
-<app-landing-page></app-landing-page>
```

Comment:
The landing page should not be removed. right?

Reply:
Since it was in the app component when you go to another page it is loaded in as well. I loved it to a route at “” so that this doesn’t happen.